resources/js/features/events/components/AchievementSet/AchievementsListItem/ProgressBarMetaText/ProgressBarMetaText.tsx

```diff
@@ -0,0 +1,63 @@
+import type { FC } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { cn } from '@/common/utils/cn';
+import { formatPercentage } from '@/common/utils/l10n/formatPercentage';
+
+interface ProgressBarMetaTextProps {
+  achievement: App.Platform.Data.Achievement;
+  playersTotal: number;
+}
+
+export const ProgressBarMetaText: FC<ProgressBarMetaTextProps> = ({
+  achievement,
+  playersTotal,
+}) => {
+  const { t } = useTranslation();
+
+  const unlocksHardcoreTotal = achievement.unlocksHardcoreTotal ?? 0;
+  const unlocksTotal = achievement.unlocksTotal ?? 0;
+  const unlockHardcorePercentage = achievement.unlockHardcorePercentage ?? 0;
+
+  return (
+    <Trans
+      i18nKey="<1>{{totalUnlocks, number}}</1> <2>({{totalHardcoreUnlocks, number}})</2> of <3>{{totalPlayers, number}}</3> <4>- {{unlockHardcorePercentage}}</4> <5>unlock rate</5>"
```

Comment:
What does the "- " in the `<4>` chunk do?

If I change the values of "unlock rate" and "{{percentage}} unlock rate" earlier in the `en_US.json` file, this seems to pick up on the "{{percentage}} unlock rate" change. Does the "- " somehow indicate the replacement should grab the combined text?

Reply:
"- " is just `children` content, eg:
```html
<span>- Hello</span>
```